Content.Server/DeltaV/Paper/SignAttemptEvent.cs

```diff
@@ -0,0 +1,8 @@
+namespace Content.Server.DeltaV.Paper;
+
+/// <summary>
+/// Raised on the pen when trying to sign a paper.
+/// If it's cancelled the signature isn't made.
```

Comment:
```suggestion
/// 	Raised on the paper when trying to sign it.
/// 	If it's cancelled the signature wasn't made.
```

Reply:
It's actually raised on the pen, and the paper entity is passed as a property. I'd rather keep it as it is because it will be needed if someone decides to port the "syndicate recruiter" antag PR from delta.

Reply:
```suggestion
/// 	Raised on the pen when trying to sign a paper.
/// 	If it's cancelled the signature wasn't made.
```